Modules/Devices/test/Devices.Domain.Tests/Identities/CancelDeletionProcessTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class CancelDeletionProcessTests
+{
+    [Fact]
+    public void Cancel_deletion_process()
+    {
+        // Arrange
+        var currentDate = DateTime.Parse("2020-01-01");
+        SystemTime.Set(currentDate);
+
+        var identity = TestDataGenerator.CreateIdentity();
+        var device = new Device(identity);
+        identity.Devices.Add(device);
+
+        var deletionProcess = identity.StartDeletionProcessAsOwner(device.Id);
```

Comment:
You could create a `CreateIdentityWithApprovedDeletionProcess` method in the `TestDataGenerator` that does all this.

Reply:
The comment says "create a `CreateIdentityWithApprovedDeletionProcess` method in the **_TestDataGenerator_**